### PR TITLE
Provide better rendering of tooltips

### PR DESCRIPTION
**Pull request by whitej6**
# Closes: #4884
# What's Changed

For all tooltips to render as a bootstrap tooltip

# TODO

- [ ] Explanation of Change(s)
- [ ] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [ ] Attached Screenshots, Payload Example
- [ ] Unit, Integration Tests
- [ ] Documentation Updates (when adding/changing features)
- [ ] Example Plugin Updates (when adding/changing features)
- [ ] Outline Remaining Work, Constraints from Design

### CODE REVIEW

**Comment by whitej6** [comment]
Currently tooltips take ~3 seconds to display and are rendered as shown below without an anchor to an element
image

Update displays tooltip immediately with an anchor to the element and applies better theming to the tooltip
image

**Comment by whitej6** [comment]
I feel this PR should go to v2.1 not the next v2.0 release so it coincides with the UI updates in v2.1

File: nautobot/core/templates/inc/javascript.html [hunk 1]
```diff
@@ -27,4 +27,6 @@
     }).ajaxStop(function() {
         loading.hide();
     });
+    // Applies bootstrap tooltip to all elements with the title attribute.
+    $("[title]").tooltip();
```
Comment (HanlinMiao) [inline nautobot/core/templates/inc/javascript.html:31]: Non-blocking, but is there a way to do this without the jquery syntax?

**Review by HanlinMiao — APPROVED** [review]
Changes look good! just one minor comment

**Pull request merged**